README.md

```diff
@@ -70,7 +70,7 @@ you to submit bugs or contact us via github if there is an issue.
 
 ##Doxygen Documentation
 
-In order to generate the doxygen documentation please follow the next steps:git
+In order to generate the doxygen documentation please follow the next steps:
```

Comment:
I would consider having examples as a code or markdown on GitHub. Nobody generates Doxygen these days :-).  If we want to keep a backward compatibility contract with those few people who do use Doxygen, let's add examples to Doxygen **and** GitHub

Reply:
Agreed. But the scope of the task was specific to doxygen documentations. We can make the scope of a future PR adding support to github. Maybe we can add a script to the doxygen cmake target that changes the code tags on files to use the doxygen ones before the generation.